C.md

```diff
@@ -0,0 +1,47 @@
+# File Metrics for C
```

Comment:
I think it could be clearer to put all these markdown files into a separate docs folder.

Reply:
@mylinhdao @clemens-mw I have seen on another PR that you have already done such a thing and are disussing the naming of the folder still. Can one of you to take care of moving the `C.md` and `CPP.md` file to your new subfolder and adjusting the links to them in the `README.md` when merging? Thank you.

Reply:
Yes, see #255